.claude/skills/marketing/SKILL.md

```
@@ -0,0 +1,242 @@
+  blog-writer: marketing/blog-writer
+  youtube-scriptwriter: marketing/youtube-scriptwriter
+  linkedin-writer: marketing/linkedin-writer
+  reviewer: marketing/reviewer
+allowed-tools:
+  - Read
+  - Write
+  - Edit
+  - Glob
+  - Grep
+  - Bash
+  - Task
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)